packages/workspace/src/builders/run-commands/run-commands.impl.ts

```diff
@@ -8,16 +8,26 @@ import { exec } from 'child_process';
 import { Observable } from 'rxjs';
 import { TEN_MEGABYTES } from '@nrwl/workspace/src/core/file-utils';
 
-try {
-  require('dotenv').config();
-} catch (e) {}
+function loadEnvVars(path?: string) {
+  if (path) {
+    const result = require('dotenv').config({ path });
+    if (result.error) {
+      throw result.error;
+    }
+  } else {
+    try {
+      require('dotenv').config();
+    } catch (e) {}
```

Comment:
since it's a "hidden" default we don't throw if the user doesn't have a root `.env` file
we do throw however, if they specify explicitly and it doesn't exist

Reply:
It makes sense. 👍